raiden_libs/test/mocks/client.py

```diff
@@ -18,6 +18,7 @@
     PathsReply,
 )
 from raiden_libs.types import Address, ChannelIdentifier, T_ChannelIdentifier
+from raiden_contracts.constants import MessageTypeId
```

Comment:
is this the only reason you are pulling the contracts as a dependency?

Reply:
Yes, as suggested by Loredana, but it's already pulled as a dev dependency anyway, just moving from dev to normal dependency because of this